### ISSUE TITLE
Postman Reports Certificate Expired for Sectigo Chains

### ISSUE THREAD

**Issue by dcatcendyn** [a85/POSTMan-Chrome-Extension#8589]
**Describe the bug**
Certificates signed by Sectigo and trusted through USERTrust are reporting the error "Error: certificate has expired".  This is related to https://www.namecheap.com/blog/sectigo-ssl-certificate-root-expiration-issue

In this case the operating system and browser select the correct chain but Postman appears to have its own chain validation and incorrectly fails on the first chain that expires.

**To Reproduce**
1. Go to HTTPS URL signed by Sectigo / USERTrust Chain - I can provide a URL for testing but did not want it public
2. Simply make a GET request
3. See error

**Expected behavior**
Trust Sectigo Chains Signed by USERTrust without having to bypass certificate verification.

**Screenshots**
![Postman](https://user-images.githubusercontent.com/34167105/83763456-54da5780-a63e-11ea-806a-eba4ff5af419.jpg)
![Postman_Console_and_Postman](https://user-images.githubusercontent.com/34167105/83763466-560b8480-a63e-11ea-8612-9b44d49278ac.jpg)

**App information (please complete the following information):**
 - Postman MacOS and Windows
 - Postman Versions 7.25.1 & 7.25.2 (only ones tested)
 - OS: [e.g. MacOS 10.15.5, Windows 10]

**Additional context**
Very simple, just make an HTTPS call to any HTTPS site protected by Sectigo/USERTrust

**Comment by codenirvana** [comment]
@dcatpegs can you share a testing endpoint with us on help@getpostman.com?

**Comment by MukeshMM** [comment]
i am also getting same issue after updating postman

**Comment by codenirvana** [comment]
We are investigating this issue till then anyone who's facing this issue please disable the "**SSL certificate verification**" setting to unblock yourself.

<img width="711" alt="Screenshot 2020-06-04 at 7 34 57 PM" src="https://user-images.githubusercontent.com/4865836/83766835-854fdf00-a69a-11ea-9658-907d1db2daa9.png">

**Comment by giorod3** [comment]
that workaround worked well thank you!

**Comment by harryi3t** [comment]
@dcatpegs @MukeshMM @giorod3 
We have identified the issue and have pushed a fix on our canary channel which can be downloaded from https://www.getpostman.com/downloads/canary

We will be pushing the fix on the stable channel soon.

**Comment by dcatcendyn** [comment]
Thanks!  I have tested Canary and it was successful.  I will test stable when its released.

**Comment by numaanashraf** [comment]
We've released v7.25.3 on stable which includes a fix for this.

**Comment by felipeabou** [comment]
I am facing this issue right now. Postman says the SLL certificate is expired, but it's not:
![image](https://user-images.githubusercontent.com/78875782/135478561-221bf4e8-6cc8-46c3-8d89-82bf1b444b72.png)

**Comment by dcatcendyn** [comment]
@felipeabou - Your issuer R3 is "Let's Encrypt" which had a root expiration this week I believe.  I'm thinking Postman may have its own cert store or your OS cert store is dated and has an expired root.  But I'd open a new ticket on this as this was has been closed for over a year.

**Comment by bratanon** [comment]
Same issue for we aswell now. With R3.

**Comment by lboix** [comment]
@felipeabou @dcatpegs @bratanon this is likely related to this article : https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/

We have some developers having issues too since this morning : they are using Postman to hit our APIs (we use Let's Encrypt to issue their SSL certificates) and getting this same message "SSL Error: Certificate has expired" for each hit.

We are using the workaround to disable “SSL certificate verification” for now.

(I am using MacOs 11.6 and Postman 9.0.3)

**Comment by deyvisPedrosa** [comment]
The same for me after updating Postman to version 9.0.3 this morning.

**Comment by bratanon** [comment]
Using the app from snap. 

Postman for Linux
Version 9.0.4

**Comment by adamsommer** [comment]
I'm having the same issue today as well. I had the issue BEFORE upgrading to 9.0.3 and the issue persists AFTER upgrading to 9.0.3

This is still a problem

**Comment by numaanashraf** [comment]
We are investigating this and further updates will be posted on https://github.com/postmanlabs/postman-app-support/issues/10338
